src/index.html

```diff
@@ -5,9 +5,57 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="cart" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="Product image"
+        class="cart__image"
+      >
+
+      <div class="cart__description">
+        <div class="cart__info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="cart__code">
```

Comment:
```suggestion
        <p class="cart__code">
```
Consider using a semantic tag for the text